### ISSUE TITLE
Low Graphic - Floating Mushrooms

### ISSUE THREAD

**Issue by ChrisD2012** [jasonsurratt/knockback-tickets#35]
**Describe the bug**
Low graphic settings, activates floating mushrooms. On the landing level, run around on the low graphics,  mushrooms float, and randomly disappear.

**To Reproduce**
1. Set lowest graphics
2. Start game 
3. run over to some mushrooms, they are not attached to the land

**Expected behavior**
Well sir I expect my mushrooms to be on the ground not floating will nilly like birds, birds are in the sky dear sir, not growing out the ground.

**Screenshots**

**Reporter**
A reporter user name (e.g. steam name) no personal names.

**Comment by jasonsurratt** [comment]
Good catch. This will be a bear. Unity's terrain object is placing those objects and it works fine on flat ground. Ugh.
